gatsby-config.js

```diff
@@ -134,8 +134,12 @@ module.exports = {
                                 path: "/edge/edge-network/tutorials",
                                 pages: [
                                     {
-                                        title: "Datastream config overrides using rules",
+                                        title: "Datastream configuration overrides using rules",
                                         path: "/edge/edge-network/tutorials/datastream-config-override-rules"
+                                    },
+                                    {
+                                        title: "Send datastream configuration overrides",
```

Comment:
- Should we keep "config" for these titles to be shorter on the left menu? also rename to:`Datastream config overrides using sendEvent`
- Let's move this first since this may be more common than rules
- Path: /edge/edge-network/tutorials/datastream-config-override-event